<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSV_to_SQLiteClass</class>
 <widget class="QWidget" name="CSV_to_SQLiteClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>408</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CSV_to_SQLite</string>
  </property>
  <widget class="QPushButton" name="buttonSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save SQLite</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Separator:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFile">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>380</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSeparator">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CSV_to_SQLite.qrc"/>
 </resources>
 <connections/>
</ui>
